contrib/inventory/foreman.py

```diff
@@ -117,10 +124,27 @@ def read_settings(self):
         self.cache_path_inventory = cache_path + "/%s.index" % script
         self.cache_path_params = cache_path + "/%s.params" % script
         self.cache_path_facts = cache_path + "/%s.facts" % script
+        self.cache_path_hostcollections = cache_path + "/%s.hostcollections" % script
         try:
             self.cache_max_age = config.getint('cache', 'max_age')
         except (ConfigParser.NoOptionError, ConfigParser.NoSectionError):
             self.cache_max_age = 60
+
+        # If the FOREMAN_ORGANIZATION environment variable is set to the
+        # label of a Satellite organization, only hosts from that
```

Comment:
Also you may want to say "label (the full name, e.g: parent-child1-grandchild2)" or something like that, just to clarify you refer to that and not just the nam